komodo/show_version.py

```diff
@@ -33,8 +33,9 @@ def get_release() -> str:
             """
             No active komodo environment found.
 
-            Either run this command from an active komodo or komodoenv environment,
-            or provide an environment manifest file path with the --path option.\
+            Either run this command from an active komodo or komodoenv
+            environment, or provide an environment manifest file path with the
+            --manifest-file option.\
```

Comment:
I did not follow up this change. Why --path -> --manifest-file?

Reply:
Håvard felt that `manifest-file` (or similar) was more explicit than `path`, and I agree, so I changed the CLI argument... but forgot to change this mention of it.